src/mp-instance.js

```diff
@@ -157,15 +167,21 @@ export default function mParticleInstance(instanceName) {
             delete instance._Store;
         }
         instance._Store = new Store(config, instance);
-        instance._Store.isLocalStorageAvailable = instance._Persistence.determineLocalStorageAvailability(
-            window.localStorage
-        );
+
```

Comment:
Just to be clear, this method is just for the tests.  i don't think there is harm in always clearing persistence, but we can discuss since I haven't actually run the tests.  

I'm just trying to avoid any potential debugging rabbit holes by being overly aggressive on reseting, rather than potentially forgetting to set the flag during a test.  This is something we can add a comment for also.